lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -5395,8 +5395,7 @@ void PatchInOutImportExport::exportShadingRate(llvm::Value *shadingRate, llvm::I
   IRBuilder<> builder(*m_context);
   builder.SetInsertPoint(insertPos);
 
-  // TODO: Current shading rate interpretation is based on GFX10.3.
-  assert(m_gfxIp.major == 10 && m_gfxIp.minor == 3);
+  assert(m_gfxIp >= GfxIpVersion({10, 3})); // Must be GFX10.3+
```

Comment:
nit: you don't need the parens around `{...}`

Reply:
Done. Thank you.

Reply:
```
Oh. I forgot I encountered this issue. In assert(),the compilation will fail unless we adopt any of these:
assert(gfx >= GfxIpVersion({10, 3})
assert(gfx >= (GfxIpVersion{10, 3}))
```

Reply:
Ah, that's unfortunate. Would it make sense to add a constructor to this struct so that we can do `GfxIpVersion(10, 3)`? (as a separate change, not in this PR)

Reply:
Sure. I will make it.